docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -183,25 +184,26 @@ details.
 Need more workers? We just add more units:    
 
 ```shell
-juju add-unit kubernetes
+juju add-unit kubernetes-worker
 ```
 
 Or multiple units at one time:  
 
 ```shell
-juju add-unit -n3 kubernetes
+juju add-unit -n3 kubernetes-worker
 ```
 You can also ask for specific instance types or other machine-specific constraints. See the [constraints documentation](https://jujucharms.com/docs/stable/reference-constraints) for more information. Here are some examples, note that generic constraints such as `cores` and `mem` are more portable between clouds. 
 
 ```shell
-juju add-unit kubernetes --constraints="instance-type=c4.large" 
-juju add-unit kubernetes --constraints="cores=4,mem=32G" 
+juju add-unit kubernetes-worker --constraints="instance-type=c4.large" 
```

Comment:
Constraints are no longer a part of `add-unit`, need to use `juju set-constraints` then `add-unit`